public/js/p3/widget/GenomeNameSelector.js

```diff
@@ -3,48 +3,48 @@ define([
   'dojo/store/JsonRest', 'dojo/dom-construct', 'dijit/TooltipDialog',
   'dojo/on', 'dijit/popup', 'dojo/_base/lang', 'dojo/dom-construct',
   'dijit/form/CheckBox', 'dojo/string', 'dojo/when', 'dijit/form/_AutoCompleterMixin',
-  '../util/PathJoin'
+  '../util/PathJoin', 'dojo/request', 'dojo/store/Memory'
 ], function (
   FilteringSelect, declare,
   Store, domConstr, TooltipDialog,
   on, popup, lang, domConstr, Checkbox,
   string, when, AutoCompleterMixin,
-  PathJoin
+  PathJoin, request, Memory
 ) {
 
   return declare([FilteringSelect, AutoCompleterMixin], {
-    apiServiceUrl: window.App.dataAPI,
+    apiServiceUrl: window.App.dataServiceURL,
     promptMessage: 'Genome name.',
     missingMessage: 'Specify genome name.',
     placeHolder: 'e.g. Mycobacterium tuberculosis H37Rv',
     searchAttr: 'genome_name',
     extraSearch: ['genome_id'],
     queryExpr: '*${0}*',
     queryFilter: '',
-    resultFields: ['genome_id', 'genome_name', 'strain', 'public', 'owner', 'reference_genome'],
+    resultFields: ['genome_id', 'genome_name', 'strain', 'public', 'owner', 'reference_genome', 'taxon_id'],
     includePrivate: true,
     includeOtherPublic: true,
     referenceOnly: true,
+    // TODO: make this defaulted to false
```

Comment:
Do we need to change the defaulted value to False before we merge?

Reply:
@hyoo The comment itself is outdated. Fixing. Thanks.